.github/workflows/lychee.yml

```diff
@@ -0,0 +1,24 @@
+name: Lychee external linkcheck
+on:
+  push:
+    branches:
+      - main  
+  pull_request:
+  schedule:
+    - cron: "18 7 * * *"  
+  workflow_dispatch: 
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Lychee Link Checker
+        uses: lycheeverse/lychee-action@v1
+        with:
+          args: --config lychee.toml ./docs
```

Comment:
Should we think about adding configuration for the `_snippets` directory too?

Reply:
Yes. i has trouble getting that to parse satisfactorily initially, so I left it off for now. I think there are some settings to tweak that will make it work